### ISSUE TITLE
Better gamepad support?

### ISSUE THREAD

**Issue by rmenessec** [spacechase0/CarryChest#2]
After some testing, I determined that Carry Chest does not play well with gamepads.

First, I didn't realize that Carry Chest requires an empty action bar slot. There doesn't appear to be a way to empty one of the slots while keeping it selected. The gamepad also doesn't have an equivalent to ability to ask SDV to select a _specific_ slot, whereas mouse and keyboard control offers the hotkeys (1-9, 0, -, =) and of course direct selection with the mouse.

In addition to the problem of activating an empty slot, I wasn't able to invoke Carry Chest any way other than left-clicking a target chest with the mouse. The equivalent gamepad action (default X button for XInput2 gamepads on PC) just "thumps" the target chest.

In short, to work properly with gamepads, Carry Chest needs two things:

- Ability to drop chests into any available inventory slot, or at least any empty action bar slot
- Gamepad support. Carry Chest doesn't seem to recognize or hook gamepad inputs at all.

Host is running Windows 10 Enterprise x64. SDV is 1.3.28 with SMAPI 2.7-stable.

I'd be happy to provide any additional information you need. Thanks!
